ROS/controls/src/nuc_motor_controls.h

```diff
@@ -0,0 +1,63 @@
+#include "ros/ros.h"
+#include "controls/MotorOrientation.h"
+#include "controls/MotorState.h"
+#include "geometry_msgs/Point.h"
+
+#include <vector>
+#include <cmath>
+
+class MotorControls {
+
+private:
+
+    /*
+     *@param pi_nh is the node for ros that interfaces with the pi
+     *@param desired_nh is the node for ros that interfaces with perceptrion
+     *@param current_sub subscribes to the current motor orientation from the pi
+     *@param desired_sub subscribes to the desired motor orientation from the publishers
+     *@param current_pub publishes to the raspberry pi the desired motor orientation
+     */ 
+    ros::NodeHandle nh;
+    // ONLY ONE NODE HANDLE PER ROS SCRIPT
```

Comment:
It's possible to have more than one node handle per script, and essential in some cases, but typically just one is used.